portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,56 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.Arrays;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private static final ArrayList<String> comments = new ArrayList<String>();
+  private static final Gson gson = new Gson();
+  private static StringBuffer sb;
```

Comment:
You probably don't need to hang on to this `StringBuffer`. You create a new one every time in `doPost` (which is necessary, since you don't want to retain state over calls to `doPost`), but it's only ever used in `doPost`. So it's safe to just keep this as a local variable within `doPost`.